qa/tasks/cephfs/test_fstop.py

```diff
@@ -17,11 +52,63 @@ def test_fstop_non_existent_cluster(self):
             pass
         else:
             raise RuntimeError('expected cephfs-top command to fail.')
-        self.mgr_cluster.mon_manager.raw_cluster_cmd("mgr", "module", "disable", "stats")
 
     def test_fstop(self):
-        self.mgr_cluster.mon_manager.raw_cluster_cmd("mgr", "module", "enable", "stats")
-        self.mount_a.run_shell(['cephfs-top',
-                                '--id=admin',
-                                '--selftest'])
-        self.mgr_cluster.mon_manager.raw_cluster_cmd("mgr", "module", "disable", "stats")
+        try:
+            self.mount_a.run_shell(['cephfs-top',
+                                    '--id=admin',
+                                    '--selftest'])
+        except CommandFailedError:
+            raise RuntimeError('cephfs-top --selftest failed')
+
+    def test_dump(self):
+        """
+        Tests 'cephfs-top --dump' output is valid
+        """
+        def verify_fstop_metrics(metrics):
```

Comment:
Shouldn't the `test_dump` handle/check multiple filesystems, as now it is supported?

Reply:
We could do that. But it's not necessary just for doing a basic validation of fstop metrics/json. Considering the `CephFSTestCase` creates only 1 filesystem by default, it's more appropriate to test what's already existing gives. More metrics tests are yet to be written on https://tracker.ceph.com/issues/58057.

Reply:
Okay, If there is tracker to take care of more metrics test we can have it there. And this is just for testing --dump command, no need to add multi-fs .